setup.cfg

```diff
@@ -11,7 +11,7 @@ python_classes =
 [flake8]
 exclude = .tox,.git,*/migrations/*,*/static/CACHE/*,docs,.ropeproject,.hypothesis
 max-line-length = 150
-ignore = E501,402
+ignore = E501,E402
```

Comment:
Newer versions of flake8 only recognise codes that match this pattern (single letter, three digits). I assume that `402` was a typo for `E402` [Module level import not at top of file](https://www.flake8rules.com/rules/E402.html) (seems like a common warning to disable).